src/Laravel/config/api-platform.php

```diff
@@ -62,7 +62,14 @@
     'graphql' => [
         'enabled' => false,
         'nesting_separator' => '__',
-        'introspection' => ['enabled' => true]
+        'introspection' => ['enabled' => true],
+        // 'middleware' => null
+    ],
+
+    'graphiql' => [
```

Comment:
@soyuka Do you want this to default to enabled when the `graphql` endpoint defaults to disabled?